casper/src/test/scala/coop/rchain/casper/api/ExploratoryDeployAPITest.scala

```diff
@@ -154,20 +153,20 @@ class ExploratoryDeployAPITest
   }
 
   it should "exploratoryDeploy return error on bonded validator" in {
-    implicit val blockDagStorage = mock[BlockDagStorage[Task]]
-    implicit val blockStore      = mock[BlockStore[Task]]
-    implicit val runtimeManager  = mock[RuntimeManager[Task]]
-    implicit val log             = mock[Log[Task]]
-    implicit val sp              = mock[Span[Task]]
+    implicit val blockDagStorage = mock[BlockDagStorage[IO]]
+    implicit val blockStore      = mock[BlockStore[IO]]
+    implicit val runtimeManager  = mock[RuntimeManager[IO]]
+    implicit val log             = mock[Log[IO]]
+    implicit val sp              = mock[Span[IO]]
 
     for {
-      result <- exploratoryDeploy[Task](
+      result <- exploratoryDeploy[IO](
                  "new return in { return!(1) }",
                  ByteString.EMPTY,
                  ValidatorIdentity(keys.head._1).some
                )
     } yield {
-      result shouldBe 'left
+      result shouldBe Symbol("left")
```

Comment:
Why it is needed? Scala 2.13 support symbol literal
```scala
Symbol("left") == 'left 
res1: Boolean = true
```

Reply:
Iirc there is a warning thrown by compiler in this case, and there is a compiler option to treat warnings as fatal, so it did not compile